### DISCUSSION TITLE
Multiple Static Sources with single calendar?

### DISCUSSION THREAD

**Discussion by jamesonuk** [mampfes/hacs_waste_collection_schedule#1801] [General]
Am I right in thinking that a static source can only have one type?
I have three different schedules setup which works but these then appear as three separate calendars.

Is there anyway I can combine these into a single calendar?

**Comment by 5ila5** [comment]
There currently is no way to combine two sources to one calendar entity. But I just created a PR (#1810) which would add support for this using a "multiple" wrapper source. This is not yet merged and might need some refinement. If you want to try it, you could download the python file from the PR and put it in the correct folder and modify your configuration according to the documentation, added by the PR.

**Reply by 5ila5** [reply]
#1810 is merged now.

Documentation on how to use the multiple source can be found [here](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/multiple.md).

This will be part of release 1.46.0 or is available now as master (git) version (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)

**Reply by danbayliss** [reply]
Is there anyway to name the multiple calendar? At the moment it's named as "Multiple Sources"

**Reply by 5ila5** [reply]
Multiple is basically just a normal source, so you can use the [calendar_title](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/installation.md#configuring-sources) Argument like for every other source

**Reply by danbayliss** [reply]
All sorted, thank you :)
